src/strategy.coffee

```diff
@@ -13,19 +13,26 @@ module.exports = (oAuth2Strategy) ->
       options.authorizationURL ?= constants.defaultAuthorizationURL
       options.tokenURL ?= constants.defaultTokenURL
       options.verifyURL ?= constants.defaultVerifyURL
+      options.scopes ?= constants.scopes
       @_verifyURL = options.verifyURL
 
       super(options, @_verifyOAuth2)
       @_oauth2.useAuthorizationHeaderforGET(true)
 
       @name = 'eveonline'
 
+    authorizationParams: (options) ->

```

Comment:
This commit seems interwoven with the previous one.  Once everything is fixed it's probably best to squash them into one commit.